docs/en_US/guides/troubleshooting.md

```diff
@@ -37,6 +37,12 @@ extra_contributors:
 1. Hold down the Power button and the Volume Down button for immediately after for 5 seconds
 1. Release the Power button but keep holding down the Volume Down button for another 5 seconds
 
+::: warning
+
+Take note that USB-C to Lightning cables are not capable of entering DFU mode. In this case, you need to use a USB-A to Lightning cable.
```

Comment:
This isn't very accurate, there's unofficial USB-C Lightning cables (as well as select official ones - though these are very rare).

What I'd suggest doing is a thing similar to the AMD notice on the Odysseyra1n Linux and Odysseyn1x pages instead (e.g. "If you are using a USB-C to Lightning cable, you will likely run into issues. If you do run into issues, you should use a USB-A to Lightning cable to follow these steps instead."

Reply:
Alrighty, let me change that.